server/tsconfig.json

```diff
@@ -31,7 +31,7 @@
     // "baseUrl": "./",                                  /* Specify the base directory to resolve non-relative module names. */
     // "paths": {},                                      /* Specify a set of entries that re-map imports to additional lookup locations. */
     // "rootDirs": [],                                   /* Allow multiple folders to be treated as one when resolving modules. */
-    // "typeRoots": [],                                  /* Specify multiple folders that act like './node_modules/@types'. */
+    "typeRoots": [ "src" ],                                  /* Specify multiple folders that act like './node_modules/@types'. */
```

Comment:
I assume you need this for your passport types. However, with this addition, I cannot run `prisma db seed`, as it complains:
```
prisma/seed.ts:175:5 - error TS2580: Cannot find name 'process'. Do you need to install type definitions for node? Try `npm i --save-dev @types/node`.
```

It seems like setting `typeRoots` means *only* packages in `typeRoots` are included. Can you get what you need by using "include"? At least, in the DoenetML repository, that seems to be the main configuration used in `tsconfig.json`.